README.md

```diff
@@ -0,0 +1,97 @@
+
+```
+var successCallback = function(data) {
+  // Display results on the page.
+};
+
+function fetchResults(searchTerm) {
+  $.ajax({
+    method: 'GET',
+    url: 'http://www.googleapis.com/books/v1/volumes?q=' + searchTerm,
+    success: successCallback,
+  });
+}
+
+fetchResults('peanut butter sandwich');
+```
+
+So let's look at this code. The first var declaration creates a callback function: `successCallback`. Then we define a function `fetchResults` that contains a `jQuery.ajax()` request. The jquery ajax method takes an object with some options set: the first two set up the request itself by specifying the url and the method. Then there is an additional parameter `success`. Here is where we link in our callback functions.
+
+Just to show you an alternate syntax for writing callbacks, we coould have achieved the same effect by using the higher order `jQuery.get()` method. As you may have guessed, this function performs an ajax GET request, so you don't have to set the request method manually. The other difference, however, is that the `$.get()` method takes a callback function as its second argument. You'll actually see this syntax much more frequently. Here's how that code would look:
+
+```
+function fetchResults(searchTerm) {
+  var url = 'http://www.googleapis.com/books/v1/volumes?q=' + searchTerm;
+  $.get(url, function(data) {
+     // Display results on the page. 
+  });
+}
+
+fetchResults('peanut butter sandwich');
+```
+
+Regardless of style, both code snippets achieve the same result by assigning the callback: now, once the ajax request returns back either sucessfully or with an error, the appropriate callback function will be called. The beauty of this is that the (asynchronous) process that occurs on the server can take as long as it needs to. Obviously, we don't want it to take a long time since that would annoy the user who is no doubt impatiently waiting for the results, but our program can now handle the indeterminate time-gap of this common asynchronous process because the process that is fetching the results now knows how to "call back" the process that will display the results once they are ready.
+
+So now you know how to explain callbacks. Essentially, callbacks are functions that, as one programner succinctly put it, "wrap or encapsulate the continuation of [our] program" after an asychronous process completes at some indeterminate point in the future.[^1]
+
+## Resources
+
+* Excellent longer analysis of async in JS: Kyle Simpson,*You Don't Know JS: Async & Performance*, Chs. [1](https://github.com/getify/You-Dont-Know-JS/blob/master/async%20&%20performance/ch1.md) and [2](https://github.com/getify/You-Dont-Know-JS/blob/master/async%20&%20performance/ch2.md)
+
+[^1]: Kyle Simpson, Kyle Simpson, ["Chapter 2: Callbacks"](https://github.com/getify/You-Dont-Know-JS/blob/master/async%20&%20performance/ch1.md) in *You Don't Know JS: Async & Performance*.
```

Comment:
Looks like github markdown may not support footnotes. https://github.com/github/markup/issues/498